internal/provider/datasource_rediscloud_subscription_test.go

```diff
@@ -53,6 +53,7 @@ data "rediscloud_payment_method" "card" {
 data "rediscloud_cloud_account" "account" {
   exclude_internal_account = true
   provider_type = "AWS" 
+  name = "PM"
```

Comment:
This would force all tests to only be run on production - they should be written so that they don't assume anything about the account they are being run in

Reply:
Yes... We actually have three options:
1. Use the cloud account 1 
2. Create cloud accounts on the fly
3. Make some environment variable that changes the `name` according to the environment.

We need make a decision on which one to go for. Thoughts?

Reply:
I'd either go with 1 - as anyone wanting to contribute to the provider is going to have that cloud account available - or have a cloud account ID environment variable